src/components/FishTankSwitch.vue

```diff
@@ -116,18 +116,18 @@ export default Vue.extend({
 })
 </script>
 
-<style lang="scss">
+<style module lang="scss">
   @import '../styles/mixins';
   @import "../../node_modules/@fishtank/colors/dist/index";
   @import "../../node_modules/@fishtank/type/dist/index";
 
 
 
-  body.user-is-tabbing input:focus + .input-checkbox-wrap {
+  body.user-is-tabbing input:focus + .inputCheckboxWrap {
     box-shadow: 0 0 0 2px $color-selected;
   }
 
-  .switch .input-checkbox-label {
+  .switch .inputCheckboxLabel {
```

Comment:
We'd like to tackle updating styles to specificity of 10. Because the class names are scoped to the component, we don't need to nest.